integration-test/1794-demote-early-landcover.py

```diff
@@ -0,0 +1,319 @@
+
+    def test_sports_cente(self):
+        self._check_min_zoom({'leisure': 'sports_centre'}, 'sports_centre', 12)
+
+    def test_recreation_ground(self):
+        self._check_min_zoom(
+            {'landuse': 'recreation_ground'}, 'recreation_ground', 12)
+
+    def test_recreation_track(self):
+        self._check_min_zoom({'leisure': 'track'}, 'recreation_track', 12)
+
+    def test_wastewater_plant(self):
+        self._check_min_zoom(
+            {'man_made': 'wastewater_plant'}, 'wastewater_plant', 12)
+
+    def test_caravan_site(self):
+        self._check_min_zoom({'tourism': 'caravan_site'}, 'caravan_site', 12)
+
+    def test_camp_site(self):
+        self._check_min_zoom({'tourism': 'camp_site'}, 'camp_site', 12)
+
+    def test_aquarium(self):
+        self._check_min_zoom({'tourism': 'aquarium'}, 'aquarium', 12)
+
+    ######################################################################
+    #
+    # zoom 13+
+    #
+    ######################################################################
+
+    def test_petting_zoo(self):
+        self._check_min_zoom({'zoo': 'petting_zoo'}, 'petting_zoo', 13)
+
+    def test_playground(self):
+        self._check_min_zoom({'leisure': 'playground'}, 'playground', 13)
+
+    def test_substation(self):
+        self._check_min_zoom({'power': 'substation'}, 'substation', 13)
+
+    def test_allotments(self):
+        self._check_min_zoom({'landuse': 'allotments'}, 'allotments', 13)
+
+    def test_pedestrian(self):
+        self._check_min_zoom({'highway': 'pedestrian'}, 'pedestrian', 13)
+
+    def test_fuel(self):
+        self._check_min_zoom({'amenity': 'fuel'}, 'fuel', 13)
+
+    def test_bridge(self):
+        self._check_min_zoom({'man_made': 'bridge'}, 'bridge', 13)
+
+    def test_footway(self):
+        self._check_min_zoom({'highway': 'footway'}, 'footway', 13)
+
+    def test_tower(self):
+        self._check_min_zoom({'man_made': 'tower'}, 'tower', 13)
+
+    def test_apron(self):
+        self._check_min_zoom({'aeroway': 'apron'}, 'apron', 13)
+
+    def test_hedge(self):
+        self._check_min_zoom({'barrier': 'hedge'}, 'hedge', 13)
+
+    def test_library(self):
+        self._check_min_zoom({'amenity': 'library'}, 'library', 13)
+
+    def test_theatre(self):
+        self._check_min_zoom({'amenity': 'theatre'}, 'theatre', 13)
+
+    def test_breakwater(self):
+        self._check_min_zoom({'man_made': 'breakwater'}, 'breakwater', 13)
+
+    def test_picnic_site(self):
+        self._check_min_zoom({'tourism': 'picnic_site'}, 'picnic_site', 13)
+
+    def test_dog_park(self):
+        self._check_min_zoom({'leisure': 'dog_park'}, 'dog_park', 13)
+
+    def test_cinema(self):
+        self._check_min_zoom({'amenity': 'cinema'}, 'cinema', 13)
+
+    def test_water_park(self):
+        self._check_min_zoom({'leisure': 'water_park'}, 'water_park', 13)
+
+    def test_harbour(self):
+        self._check_min_zoom({'landuse': 'harbour'}, 'harbour', 13)
```

Comment:
I see a test for `harbour`, but looks like we need to modify the data clamp to 13 (from 9)? Discuss.

Reply:
The test was just to check - `harbour` was already [clamped to z13](https://github.com/tilezen/vector-datasource/pull/1797/files/1c15d82b1c14d8718d6658c73879e8099eb39978#diff-a2a00baa9c8d3ef5107119155e3872ceR777).